app/bingey-api/utils/reader.js

```diff
@@ -0,0 +1,64 @@
+// *** USAGE ***
+// Upload a data.tsv file to api utils directory
+// Run yarn import to upsert data from the file to the database
```

Comment:
This should be `yarn import-data`. The `yarn import` command is used by Yarn itself to generate the `.lock` file. 
Maybe these instructions should be added to the _readme_ too?